config/config.go

```diff
@@ -351,6 +352,12 @@ func validateConfig(c *Config) error {
 		return errors.New("'access_log_level' must be set to either \"none\" or \"all\"")
 	}
 
+	for categoryName := range c.MetricCategories {
+		if categoryName != strings.ToLower(categoryName) {
+			return fmt.Errorf("Names in 'metric_categories' must be in lower case: %s", categoryName)
```

Comment:
nitpick: reword to `"metric_categories names must be lowercase: %q", categoryName`